olm/quickstarts/rhosak-openshift-connect-quickstart.yaml

```
@@ -3,160 +3,336 @@ kind: ConsoleQuickStart
 metadata:
   name: rhosak-devsandbox-connect-cli-toolscontainer-quickstart
 spec:
-  displayName: Connecting Red Hat OpenShift Streams for Apache Kafka to OpenShift
+  displayName: Connecting OpenShift to Red Hat OpenShift Streams for Apache Kafka using the CLI
   tags:
     - streams
     - kafka
   durationMinutes: 10
-  description: Using the OpenShift Streams for Apache Kafka cloud service in your OpenShift cluster
+  description: Connecting an OpenShift project to Red Hat OpenShift Streams for Apache Kafka using the RHOAS CLI
   prerequisites:
-    - Access to Red Hat OpenShift Streams for Apache Kafka (for more information, visit https://cloud.redhat.com/application-services).
-    - A running OpenShift Streams for Apache Kafka instance.
-  introduction: >-
-    ### This quick start shows you how to connect Red Hat OpenShift Streams for Apache Kafka to OpenShift using the RHOAS CLI.
+    - You have a Red Hat account.
+    - You’ve created a Kafka instance in [Streams for Apache Kafka](https://cloud.redhat.com/beta/application-services/streams/) and the instance is in the **Ready** state.
```

```suggestion
    - You’ve created a Kafka instance in [Streams for Apache Kafka](https://console.redhat.com/beta/application-services/streams/) and the instance is in the **Ready** state.
```
console.redhat.com is already working, should we switch to it?
cc @pmuir

---

as this is Work in Progress, I don't see why not.